packages/waku/src/lib/middleware/rsc.ts

```diff
@@ -42,7 +42,10 @@ export const rsc: Middleware = (options) => {
           contentType: headers['content-type'] || '',
         };
         const readable = await (ctx.devServer
-          ? ctx.devServer.renderRscWithWorker(args)
+          ? ctx.devServer.renderRscWithWorker({
+              ...args,
+              initialModules: ctx.devServer.initialModules,
+            })
```

Comment:
Let's change the signature:
```suggestion
          ? ctx.devServer.renderRscWithWorker(args, {
              initialModules: ctx.devServer.initialModules,
            })
```